core-java-modules/core-java-lambdas/src/test/java/com/baeldung/anonymousclass/AnonymousClassToLambdaIntegrationTest.java

```diff
@@ -0,0 +1,34 @@
+package com.baeldung.anonymousclass;
+
+import static org.junit.Assert.assertEquals;
+
+import org.junit.Test;
+
+public class AnonymousClassToLambdaIntegrationTest {
+
+    @Test
+    public void whenPassingAnonymousClass_thenSuccess() {
+        final SenderService emailSenderService = new EmailSenderService();
+
+        final String emailNotif = emailSenderService.callSender(new Sender() {
+            @Override
+            public String send(String message) {
+                return message;
+            }
+        });
+
+        assertEquals(emailNotif, "Email Notification");
+    }
+
+    @Test
+    public void whenPassingLambdaExpression_thenSuccess() {
+        final SenderService smsSenderService = new SmsSenderService();
+
+        final String smsNotif = smsSenderService.callSender((String message) -> {
+            return message;
+        });
```

Comment:
You can simplify this statement with just an expression:
`        final String smsNotif = smsSenderService.callSender((String message) -> message);`

Reply:
Yes, I did it on purpose to explain in the article that this can be simplified to: 
`final String smsNotif = smsSenderService.callSender((String message) -> message);`